gobblin-hive-registration/src/main/java/gobblin/hive/HivePartition.java

```diff
@@ -45,6 +45,11 @@ private HivePartition(Builder builder) {
     this.values = ImmutableList.<String> copyOf(builder.values);
   }
 
+  @Override
+  public String toString() {
+    return this.values.toString();
```

Comment:
You probably want to use `ArrayUtils.toString(list.toArray())` (as it is, you will get the hashCode instead).